pkgs/os-specific/linux/evdi/default.nix

```diff
@@ -26,12 +26,17 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-xB3AHg9t/X8vw5p7ohFQ+WuMjb1P8DAP3pROiwWkVPs=";
   };
 
-  env.NIX_CFLAGS_COMPILE = toString [
+  env.CFLAGS = toString [
     "-Wno-error"
     "-Wno-error=discarded-qualifiers" # for Linux 4.19 compatibility
```

Comment:
We don't have Linux 4.19 any more, so hopefully would be possible to clean these up?